test/test-fs-event.c

```diff
@@ -53,6 +53,12 @@ static char fs_event_filename[PATH_MAX];
 static char fs_event_filename[1024];
 #endif  /* defined(PATH_MAX) */
 static int timer_cb_touch_called;
+static int timer_cb_exact_called;
+
+static void fs_event_fail(uv_fs_event_t* handle, const char* filename,
```

Comment:
Sure. I just moved this from the bottom of the file. I was getting not defined yet errors.